cmd/snap/cmd_list.go

```diff
@@ -52,16 +53,31 @@ func (s snapsByName) Less(i, j int) bool { return s[i].Name < s[j].Name }
 func (s snapsByName) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
 func (x *cmdList) Execute([]string) error {
+	if x.Refresh {
+		return listRefreshes()
+	}
 	return listSnaps(x.Positional.Snaps)
 }
 
+func listRefreshes() error {
```

Comment:
Instead of duplicating listSnaps, I suggest passing it a `*client.ListOptions` as an argument.